src/sys/wait.rs

```diff
@@ -207,7 +233,7 @@ impl WaitStatus {
     }
 }
 
-pub fn waitpid<P: Into<Option<Pid>>>(pid: P, options: Option<WaitPidFlag>) -> Result<WaitStatus> {
+pub fn waitpid<P: Into<WaitTarget>>(who: P, options: Option<WaitPidFlag>) -> Result<WaitStatus> {
```

Comment:
Can you provide docs for `waitpid` here? I think it's important to note in them that passing a straight `Pid` translates to `WaitTarget::Child(Pid)`

Reply:
You can reference #645 so it should be pretty easy.